src/App.css

```diff
@@ -1,38 +1,243 @@
-  }
-  to {
-    transform: rotate(360deg);
-  }
+/*
+  7. Remove built-in form typography styles
+*/
+input,
+button,
+textarea,
+select {
+	font: inherit;
+}
+
+/*
+  8. Avoid text overflows
+*/
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+	overflow-wrap: break-word;
+}
+
+/*
+  9. Create a root stacking context
+*/
+#root,
+#__next {
+	isolation: isolate;
+}
+
+html,
+body {
+	height: 100%;
+	line-height: 2;
+	font-family: 'Oswald', sans-serif;
+	background-color: hsl(0, 0%, 100%);
+	text-transform: uppercase;
+}
+
+.wrapper {
+	max-width: 1280px;
+	width: 80%;
+	margin: 0 auto;
+}
+
+header {
+	background: center/cover no-repeat url(../assets/arnold.jpg);
+	min-height: min(80vh, 500px);
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	margin-bottom: 32px;
+}
+
+.mainHeading {
+	color: gold;
+	text-align: center;
+	font-family: 'Anton', sans-serif;
+	font-size: 6rem;
+	text-shadow: 2px 2px 2px hsl(0, 0%, 0%);
+}
+
+form {
+	display: flex;
+	flex-direction: column;
+}
+
+.selectContainer {
+	text-align: center;
+	margin-bottom: 16px;
+}
+
+.selectContainer label {
+	font-weight: 500;
+}
+
+select {
+	background-color: hsl(0, 0%, 100%);
+	border: 2px solid hsl(51, 100%, 50%);
+	border-radius: 16px;
+	padding: 0.25rem 0.75rem;
+}
+
+select:hover {
+	color: hsl(51, 100%, 50%);
+	background-color: hsl(0, 0%, 0%);
+}
+
+select:hover,
+select:focus-visible {
+	cursor: pointer;
+}
+
+.btnReset {
+	margin: 16px;
+}
+
+label {
+	margin: 0 0.5rem;
+}
+
+fieldset {
+	position: relative;
+	text-align: center;
+	border: 2px solid hsl(51, 100%, 50%);
+	border-radius: 16px;
+	margin-bottom: 16px;
+}
+
+fieldset > legend {
+	font-weight: 500;
+}
+
+button {
+	color: hsl(0, 0%, 0%);
+	background-color: hsl(0, 0%, 100%);
+	padding: 0.25rem 0.75rem;
+	outline: 2px solid hsl(51, 100%, 50%);
+	border: none;
+	border-radius: 16px;
+	text-transform: uppercase;
+}
+
+button:hover,
+button:focus-visible {
+	cursor: pointer;
+	color: hsl(51, 100%, 50%);
+	background-color: hsl(0, 0%, 0%);
+}
+
+.btnGenerate {
+	align-self: center;
+	width: fit-content;
+	font-size: 2rem;
+	font-weight: 700;
+	margin: 16px 0;
+}
+
+.workout {
+	margin-top: 32px;
+	margin-bottom: 32px;
+}
+
+.workoutList {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	row-gap: 4rem;
+	column-gap: 1.5rem;
+	list-style: none;
+	padding: 16px;
+	text-align: center;
+	border-radius: 16px;
+	/* outline: 2px solid gold; */
+}
+
+.workoutList li {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	align-items: center;
+	flex-basis: 30%;
+	/* outline: 1px solid red; */
+}
+
+img {
+	max-height: 150px;
+	max-width: auto;
+}
+
+.exerciseName {
+	text-transform: uppercase;
+}
+
+.footer {
+	width: 100%;
+	text-align: center;
+	background-color: hsl(0, 0%, 0%);
+	color: hsl(51, 100%, 50%);
+}
+
+@media (width < 550px) {
+	.wrapper {
+		width: 95%;
+	}
+
+	.mainHeading {
+		width: fit-content;
+		font-size: 3.5rem;
+		line-height: 1.5;
+	}
+}
+
+.lol {
```

Comment:
As long as you and team members understand this, it's all good! But for someone looking at it brand new, they might not know what this refers to!